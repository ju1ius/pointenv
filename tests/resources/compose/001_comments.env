COMMENT=notacomment # comment
NOT_COMMENT=foo# not a comment
QUOTE_COMMENT='foo'# a comment
DQ_COMMENT="foo"# a comment
