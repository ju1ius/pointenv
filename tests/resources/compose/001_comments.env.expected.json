[
  {
    "key": "COMMENT",
    "value": "notacomment"
  },
  {
    "key": "DQ_COMMENT",
    "value": "foo"
  },
  {
    "key": "NOT_COMMENT",
    "value": "foo# not a comment"
  },
  {
    "key": "QUOTE_COMMENT",
    "value": "foo"
  }
]
